Godeps/Godeps.json

```diff
@@ -10,6 +10,9 @@
 			"ImportPath": "github.com/antchfx/xpath",
 			"Rev": "1f3266e7730781347c9decd229c1fe1c18be760c"
 		},
+			"ImportPath": "github.com/dutchcoders/go-clamd",
```

Comment:
@joelsmith why you have to work in different fs namespace? is that for scanning live containers from the clam running on the host?

I would certainly prefer 1) and just add support for FILDES to go-clamd. Also I think we don't want to do FILDES scan by default as I don't  think scanning files in different FS namespace is a common use-case.

How long down the filepath.Walk() scan takes on large images?